templates/taxbrain/input_form.html

```diff
@@ -264,4 +264,22 @@ <h4 class="modal-title" id="myModalLabel">Change Start Year</h4>
       </div>
     </div>
   </div>
+  <div class="modal fade" id="data-choice-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="myModalLabel">Change Data Source</h4>
+        </div>
+        <div class="modal-body">
+          Warning: This will clear your current inputs.
+        </div>
+        <div class="modal-footer">
+          <a type="button" class="btn btn-default" data-dismiss="modal">Nevermind.</a>
+          <a type="button" class="btn btn-primary" id="data-source-link">OK, go for it.</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
 {% endblock %}
```

Comment:
I think this section opens the dialogue box like this one:

<img width="609" alt="screen shot 2018-03-02 at 5 13 49 pm" src="https://user-images.githubusercontent.com/9206065/36924564-1e5e16aa-1e3d-11e8-9e5a-8b91efe69dd3.png">

However, after adding what I think is the exact same code but with different names as the `start_year` code, I have no idea why this doesn't work but the `start_year` does.